src/index.js

```diff
@@ -27,15 +28,32 @@ app.use(cors());
 
 dotenv.config();
 const port = process.env.PORT;
+const sequelize = new Sequelize(
+  process.env.DATABASE,
+  process.env.DATABASE_USERNAME,
+  process.env.DATABASE_PASSWORD,
+  {
+    host: "postgres",
+    dialect: "postgres",
+  }
+);
+
```

Comment:
Database initialization has been already initialized. I don't why we have this here